source/loader/layers/sanitizer/asan_interceptor.cpp

```diff
@@ -708,8 +708,7 @@ ur_result_t SanitizerInterceptor::prepareLaunch(
         };
 
         // Write debug
-        // We use "uint64_t" here because EnqueueWriteGlobal will fail when it's "uint32_t"
-        uint64_t Debug = Options(logger).Debug ? 1 : 0;
+        uint32_t Debug = Options(logger).Debug ? 1 : 0;
         EnqueueWriteGlobal(kSPIR_AsanDebug, &Debug, sizeof(Debug));
```

Comment:
I think we can use Options directly, no need to use a temporary variable
```suggestion
        EnqueueWriteGlobal(kSPIR_AsanDebug, &Option(logger).Debug, sizeof(Option(logger).Debug));
```

Reply:
If the type of `Option::Debug` is not correct, we can change it at `Option`